test/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware.Tests/Logging/AcceptanceTests.cs

```diff
@@ -714,6 +718,48 @@ await RunAsync(
             });
     }
 
+#if NET9_0_OR_GREATER
+    [Fact]
+    public async Task HttpRequestBuffering()
```

Comment:
It doesn't have to be this test, but it would be nice to have some test or sample code that uses the feature the way we expect an end-user to use it. For example I would not expect end users to new up the Options type, create their own instances of HttpRequestBuffer or manually add objects to the DI container.

Reply:
Added GlobalBuffering_CanonicalUsecase

Reply:
Update this test, users (as of today) are supposed to inject `IHttpRequestBufferManager` into their middleware and call `Flush()` and/or  `FlushCurrentRequestLogs()` on it